jackson/src/test/java/com/baeldung/jackson/deserialization/nested/DeserializeWithNestedPropertiesTest.java

```diff
@@ -0,0 +1,63 @@
+package com.baeldung.jackson.deserialization.nested;
+
+import static org.junit.Assert.*;
+
+import java.io.IOException;
+
+import org.junit.Test;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+
+public class DeserializeWithNestedPropertiesTest {

```

Comment:
Let's rename this class to end with "UnitTest" instead of just "Test" so that the build will run it.